packages/inter-protocol/src/auction/auctionBook.js

```diff
@@ -88,17 +92,19 @@ export const makeOfferSpecShape = (bidBrand, collateralBrand) => {
 
 /**
  * @typedef {object} BookDataNotification
- *
- * @property {Ratio         | null}      startPrice identifies the priceAuthority and price
- * @property {Ratio         | null}      currentPriceLevel the price at the current auction tier
- * @property {Amount<'nat'> | null}      startProceedsGoal The proceeds the sellers were targeting to raise
- * @property {Amount<'nat'> | null}      remainingProceedsGoal The remainder of
- *     the proceeds the sellers were targeting to raise
- * @property {Amount<'nat'> | undefined} proceedsRaised The proceeds raised so far in the auction
- * @property {Amount<'nat'>}             startCollateral How much collateral was
- *    available for sale at the start. (If more is deposited later, it'll be
- *    added in.)
- * @property {Amount<'nat'> | null}      collateralAvailable The amount of collateral remaining
+ * @property {Ratio | null} startPrice identifies the priceAuthority and price
+ * @property {Ratio | null} currentPriceLevel the price at the current auction
+ *   tier
+ * @property {Amount<'nat'> | null} startProceedsGoal The proceeds the sellers
+ *   were targeting to raise
+ * @property {Amount<'nat'> | null} remainingProceedsGoal The remainder of the
+ *   proceeds the sellers were targeting to raise
+ * @property {Amount<'nat'> | undefined} proceedsRaised The proceeds raised so
+ *   far in the auction
+ * @property {Amount<'nat'>} startCollateral How much collateral was available
+ *   for sale at the start. (If more is deposited later, it'll be added in.)
+ * @property {Amount<'nat'> | null} collateralAvailable The amount of collateral
+ *   remaining
```

Comment:
the tool doesn't do vertical alignment. what parallel is lost? 

Note that when this is rendered into HTML it will look like the following. (using default theme)
<img width="545" alt="Screenshot 2023-07-18 at 11 36 25 AM" src="https://github.com/Agoric/agoric-sdk/assets/21505/8ee8bb57-e6ec-4bfb-8779-1245154411a5">

Reply:
When I was looking through the changes, it became clear to me that the reformatter thinks that the only version of the comments that matters (not the **primary** version, the **only** version) is what gets rendered for readers. I find the comments themselves useful at times to the maintainer (me).

The original formatting (which is what I look at to maintain it) directs my attention to the fact that there is a semantic difference between values which are `| null`, `| undefined`, and unadorned. The laid-out version makes that hard to see.

```
 * @typedef {object} BookDataNotification
 *
 * @property {Ratio         | null}      startPrice identifies the priceAuthority and price
 * @property {Ratio         | null}      currentPriceLevel the price at the current auction tier
 * @property {Amount<'nat'> | null}      startProceedsGoal The proceeds the sellers were targeting to raise
 * @property {Amount<'nat'> | null}      remainingProceedsGoal The remainder of
 *     the proceeds the sellers were targeting to raise
 * @property {Amount<'nat'> | undefined} proceedsRaised The proceeds raised so far in the auction
 * @property {Amount<'nat'>}             startCollateral How much collateral was
 *    available for sale at the start. (If more is deposited later, it'll be
 *    added in.)
 * @property {Amount<'nat'> | null}      collateralAvailable The amount of collateral remaining
 * 
 ```

Reply:
ok I understand. There no option to retain the whitespace and I don't expect there will be since one key value of the tool is normalizing whitespace. It doesn't know when it's intentional or accidental. The way to maintain whitespace is to make it a codeblock, but that can't happen within the typedef.

If this is a show-stopped for you, let's discuss higher bandwidth

Reply:
> If this is a show-stopped for you, let's discuss higher bandwidth

Not a show-stopper. On the whole, this is an improvement.

IMO, it may not be enough of an improvement to outweigh the control that it gives up, but I seem to be a minority in that viewpoint. I'm okay with that.